src/store/modules/brushModule.js

```diff
@@ -126,39 +128,18 @@ const getters = {
 /**
  * EnabledElementCallback - Element specific initilisation.
  * @public
- * @param  {Object} enabledElement  The element on which the module is
+ * @param  {Object} enabledElement - The element on which the module is
  *                                  being initialised.
+ * @returns {void}
  */
 function enabledElementCallback(enabledElement) {
   setters.elementVisible(enabledElement);
 }
 
-/**
- * RemoveEnabledElementCallback - Element specific memory cleanup.
- * @public
- * @param  {Object} enabledElement  The element being removed.
- */
-function removeEnabledElementCallback(enabledElement) {
-  if (!external.cornerstone) {
-    return;
-  }
-
-  const cornerstoneEnabledElement = external.cornerstone.getEnabledElement(
-    enabledElement
-  );
-
-  const enabledElementUID = cornerstoneEnabledElement.uuid;
-  const colormap = external.cornerstone.colors.getColormap(state.colorMapId);
-  const numberOfColors = colormap.getNumberOfColors();
-
-  // Remove enabledElement specific data.
-  delete state.visibleSegmentations[enabledElementUID];
-  delete state.imageBitmapCache[enabledElementUID];
-}
-
```

Comment:
@JamesAPetts, do you use this anywhere?

Reply:
We are currently not using it, but we should. Instead of deleting this we should add it to the export so its called by `store/internals/removeEnabledElement.js`. Then test it of course.

If OP is willing to undo the deletion of these lines, I'll hook it up and test it in a coming patch.

Reply:
Or to keep the linter happy, comment out the lines and add `// TODO -> Test this before adding it to the module.`

Reply:
I've reverted this change.